README.md

```diff
@@ -121,6 +121,56 @@ On your template, provide a button that will allow the user to log in to the app
 </button>
 ```
 
+### Preserve application state through redirects
```

Comment:
I would move this to our FAQ as I believe this is not a main feature of our SDK. We want to avoid that people rely to heavily on our SDK for tracking application state.